어드민 패키지에서 상품과 관련된 로직에서 에러가 발생했다.

## 버그 내용
> 버그에 대한 내용을 알려주세요.

```
symbol
    symbol:   method getLastModifiedAt()
    location: class UserLogRecordResponse
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:117: error: cannot find symbol
        userRepository.save(User.builder()
                                ^
  symbol:   method builder()
  location: class User
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:122: error: cannot find symbol
        profileRepository.save(Profile.builder()
                                      ^
  symbol:   method builder()
  location: class Profile
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:143: error: cannot find symbol
        saveImages(newProduct.getId(), request.getImageUrls());
                                              ^
  symbol:   method getImageUrls()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:149: error: cannot find symbol
        Product newProduct = Product.builder()
                                    ^
  symbol:   method builder()
  location: class Product
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:151: error: cannot find symbol
                .title(request.getTitle())
                              ^
  symbol:   method getTitle()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:153: error: cannot find symbol
                .productCategory(request.getProductCategory())
                                        ^
  symbol:   method getProductCategory()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:154: error: cannot find symbol
                .description(request.getDescription())
                                    ^
  symbol:   method getDescription()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:155: error: cannot find symbol
                .sellPrice(request.getSellPrice())
                                  ^
  symbol:   method getSellPrice()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:248: error: cannot find symbol
        List<String> savedImageUrls = saveImages(id, request.getImageUrls());
                                                            ^
  symbol:   method getImageUrls()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:250: error: cannot find symbol
        Product updateProduct = Product.builder()
                                       ^
  symbol:   method builder()
  location: class Product
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:253: error: cannot find symbol
                .title(request.getTitle())
                              ^
  symbol:   method getTitle()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:254: error: cannot find symbol
                .productCategory(request.getProductCategory())
                                        ^
  symbol:   method getProductCategory()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:256: error: cannot find symbol
                .description(request.getDescription())
                                    ^
  symbol:   method getDescription()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java:257: error: cannot find symbol
                .sellPrice(request.getSellPrice())
                                  ^
  symbol:   method getSellPrice()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:80: error: cannot find symbol
        ProductCreateRequest productCreateRequest = new ProductCreateRequest(request.getTitle(), request.getProductCategory(),request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                    ^
  symbol:   method getTitle()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:80: error: cannot find symbol
        ProductCreateRequest productCreateRequest = new ProductCreateRequest(request.getTitle(), request.getProductCategory(),request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                                        ^
  symbol:   method getProductCategory()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:80: error: cannot find symbol
        ProductCreateRequest productCreateRequest = new ProductCreateRequest(request.getTitle(), request.getProductCategory(),request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                                                                     ^
  symbol:   method getDescription()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:80: error: cannot find symbol
        ProductCreateRequest productCreateRequest = new ProductCreateRequest(request.getTitle(), request.getProductCategory(),request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                                                                                               ^
  symbol:   method getSellPrice()
  location: variable request of type ProductCreateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:118: error: cannot find symbol
        ProductUpdateRequest productUpdateRequest = new ProductUpdateRequest(request.getTitle(), request.getProductCategory(), request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                    ^
  symbol:   method getTitle()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:118: error: cannot find symbol
        ProductUpdateRequest productUpdateRequest = new ProductUpdateRequest(request.getTitle(), request.getProductCategory(), request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                                        ^
  symbol:   method getProductCategory()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:118: error: cannot find symbol
        ProductUpdateRequest productUpdateRequest = new ProductUpdateRequest(request.getTitle(), request.getProductCategory(), request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                                                                      ^
  symbol:   method getDescription()
  location: variable request of type ProductUpdateRequest
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/admin/presentation/AdminController.java:118: error: cannot find symbol
        ProductUpdateRequest productUpdateRequest = new ProductUpdateRequest(request.getTitle(), request.getProductCategory(), request.getDescription(), request.getSellPrice(), multipartFiles);
                                                                                                                                                                ^
  symbol:   method getSellPrice()
  location: variable request of type ProductUpdateRequest
Note: /var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/order/domain/OrderRepository.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
100 errors

> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org/

BUILD FAILED in 19s
2 actionable tasks: 2 executed
```

## 버그 상황
> 어떤 상황에서 버그가 발생했는지 알려주세요.

젠킨스에서 프론트엔드와 백엔드를 통합해서 Build & Deploy 하는 과정에서 백엔드 Build 에서 실패가 떴다.

## 참고 자료
> 참고할 만한 내용 혹은 주소가 있다면 작성해 주세요.

--- a/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/admin/application/AdminService.java
@@ -112,7 +112,7 @@ public class AdminService {
 
     // 사용자 정보 수정
     public void updateUserInfo(Long userId, UserUpdateRequest request) {
-        User user = userRepository.getByUserId(userId);
+        User user = userRepository.getById(userId);
         Profile profile = profileRepository.getByUserId(userId);
         userRepository.save(User.builder()
                 .email(user.getEmail())
@@ -162,7 +162,7 @@ public class AdminService {
         String uniqueFileName = FileUtils.generateUniqueFileName(image.getOriginalFilename());
         String savedImageUrl = s3Service.saveFile(image, uniqueFileName);
 
-        productImageRepository.save(new ProductImage(productRepository.getByProductId(productId), savedImageUrl));
+        productImageRepository.save(new ProductImage(productRepository.getById(productId), savedImageUrl));
 
         return savedImageUrl;
     }
@@ -206,7 +206,7 @@ public class AdminService {
                         return productResponse;
                     }
 
-                    User user = userRepository.getByUserId(product.getUser().getId());
+                    User user = userRepository.getById(product.getUser().getId());
                     Profile profile = profileRepository.getByUserId(product.getUser().getId());
 
                     ProductResponse productResponse = new ProductResponse(
@@ -220,19 +220,19 @@ public class AdminService {
 
     // 상품 상세 조회
     public ProductDetailResponse viewOneProduct(Long id) {
-        Product product = productRepository.getByProductId(id);
+        Product product = productRepository.getById(id);
         List<String> imageUrls = productImageRepository.findAllImageUrlByProductId(product.getId());
 
         if (product.getUser() == null) {
             return ProductDetailResponse.of(product, imageUrls);
         }
-        User user = userRepository.getByUserId(product.getUser().getId());
+        User user = userRepository.getById(product.getUser().getId());
         return ProductDetailResponse.of(product, imageUrls, user);
     }
 
     // 수정할 상품
     public ProductUpdateResponse showSelectedProduct(Long id) {
-        Product selectedProduct = productRepository.getByProductId(id);
+        Product selectedProduct = productRepository.getById(id);
         List<String> images = productImageRepository.findAllImageUrlByProductId(id);
         return new ProductUpdateResponse(selectedProduct, images);
     }
@@ -240,7 +240,7 @@ public class AdminService {
     // 상품 수정
     @Transactional
     public ProductUpdateResponse updateProduct(ProductUpdateRequest request, Long id) throws IOException {
-        Product selectedProduct = productRepository.getByProductId(id);
+        Product selectedProduct = productRepository.getById(id);
 
         deleteImageByProductId(id);
         productImageRepository.deleteByProductId(id);

--- a/backend/src/main/java/woorifisa/goodfriends/backend/order/application/OrderService.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/order/application/OrderService.java
@@ -70,8 +70,8 @@ public class OrderService {
             throw new OwnProductException();
         }
 
-        Product foundProduct = productRepository.getByProductId(request.getProductId());
-        User foundUser = userRepository.getByUserId(userId);
+        Product foundProduct = productRepository.getById(request.getProductId());
+        User foundUser = userRepository.getById(userId);
 
         Order newOrder = orderRepository.save(createOrder(foundProduct, foundUser, request));
         return newOrder.getId();
@@ -93,7 +93,7 @@ public class OrderService {
     }
 
     public boolean ownProduct(Long productId, Long userId) {
-        Product product = productRepository.getByProductId(productId);
+        Product product = productRepository.getById(productId);
         return product.getUser().getId() == userId;
     }
 
@@ -121,7 +121,7 @@ public class OrderService {
             throw new NotOwnProductException();
         }
 
-        Product product = productRepository.getByProductId(productId);
+        Product product = productRepository.getById(productId);
 
         if(product.getStatus() != ProductStatus.SELL) {
             Order order = orderRepository.findByProductIdAndConfirmStatus(productId, OrderStatus.RESERVATION);
@@ -152,14 +152,14 @@ public class OrderService {
     public UserDealResponse dealOrder(Long orderId) {
 
         Order order = orderRepository.getById(orderId);
-        Product product = productRepository.getByProductId(order.getProduct().getId());
+        Product product = productRepository.getById(order.getProduct().getId());
 
         if(product.getStatus() == ProductStatus.SELL) {
             orderRepository.updateOrderStatus(orderId, OrderStatus.RESERVATION);
             productRepository.updateProductStatus(order.getProduct().getId(), ProductStatus.RESERVATION);
         }
 
-        User user = userRepository.getByUserId(order.getUser().getId());
+        User user = userRepository.getById(order.getUser().getId());
 
         UserDealResponse response = new UserDealResponse(user.getNickname(), user.getProfileImageUrl(), user.getEmail());
 

--- a/backend/src/main/java/woorifisa/goodfriends/backend/product/application/ProductService.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/product/application/ProductService.java
@@ -57,7 +57,7 @@ public class ProductService {
     public Long saveProduct(final Long userId, final ProductCreateRequest request, final List<MultipartFile> multipartFiles) throws IOException {
         validateUser(userId);
 
-        User foundUser = userRepository.getByUserId(userId);
+        User foundUser = userRepository.getById(userId);
         ProductCreateRequest newRequest = createProductCreateRequest(request, multipartFiles);
         Product newProduct = productRepository.save(newRequest.toEntity(foundUser, newRequest));
 
@@ -107,7 +107,7 @@ public class ProductService {
         String uniqueFileName = FileUtils.generateUniqueFileName(image.getOriginalFilename());
         String savedImageUrl = s3Service.saveFile(image, uniqueFileName);
 
-        ProductImage productImage = new ProductImage(productRepository.getByProductId(productId), savedImageUrl);
+        ProductImage productImage = new ProductImage(productRepository.getById(productId), savedImageUrl);
         productImageRepository.save(productImage);
         return savedImageUrl;
     }
@@ -145,7 +145,7 @@ public class ProductService {
                     if (product.getUser() == null)
                         return ProductResponse.of(product, imageUrl);
 
-                    User user = userRepository.getByUserId(product.getUser().getId());
+                    User user = userRepository.getById(product.getUser().getId());
                     Profile profile = profileRepository.getByUserId(product.getUser().getId());
                     return ProductResponse.of(product, imageUrl, user, profile);
                 })
@@ -161,7 +161,7 @@ public class ProductService {
         if (product.getUser() == null) {
             return ProductDetailResponse.of(product, imageUrls);
         }
-        User user = userRepository.getByUserId(product.getUser().getId());
+        User user = userRepository.getById(product.getUser().getId());
         return ProductDetailResponse.of(product, imageUrls, user);
     }
 
@@ -174,7 +174,7 @@ public class ProductService {
             throw new NotAccessThisProductException();
         }
 
-        Product selectedProduct = productRepository.getByProductId(productId);
+        Product selectedProduct = productRepository.getById(productId);
         List<String> images = productImageRepository.findAllImageUrlByProductId(productId);
         return ProductUpdateResponse.of(selectedProduct, images);
     }
@@ -203,7 +203,7 @@ public class ProductService {
         if (userId == null) {
             return false;
         }
-        Product product = productRepository.getByProductId(productId);
+        Product product = productRepository.getById(productId);
         return userId.equals(product.getUser().getId());
     }
 

--- a/backend/src/main/java/woorifisa/goodfriends/backend/product/domain/ProductRepository.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/product/domain/ProductRepository.java
@@ -10,7 +10,7 @@ import woorifisa.goodfriends.backend.product.exception.NotFoundProductException;
 import java.util.List;
 
 public interface ProductRepository extends JpaRepository<Product, Long> {
-    default Product getByProductId(final Long id) {
+    default Product getById(final Long id) {
         return findById(id)
                 .orElseThrow(NotFoundProductException::new);
     }

--- a/backend/src/main/java/woorifisa/goodfriends/backend/profile/application/ProfileService.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/profile/application/ProfileService.java
@@ -43,7 +43,7 @@ public class ProfileService {
 
     @Transactional
     public ProfileDetailResponse findMyProfile(final Long userId) {
-        User user = userRepository.getByUserId(userId);
+        User user = userRepository.getById(userId);
         Profile profile = profileRepository.findByUserId(userId).orElse(null);
 
         return ProfileDetailResponse.of(user, profile);
@@ -51,7 +51,7 @@ public class ProfileService {
 
     @Transactional
     public void update(final Long userId, final ProfileUpdateRequest request) {
-        User user = userRepository.getByUserId(userId);
+        User user = userRepository.getById(userId);
         user.updateNickname(request.getNickName());
         userRepository.save(user);
 

--- a/backend/src/main/java/woorifisa/goodfriends/backend/profile/domain/Profile.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/profile/domain/Profile.java
@@ -17,7 +17,6 @@ public class Profile extends BaseEntity {
 
     private static final Pattern MOBILE_NUMBER = Pattern.compile("^\\d{3}-\\d{3,4}-\\d{4}$");
 
-    private static final Pattern MOBILE_NUMBER = Pattern.compile("^\\d{3}-\\d{3,4}-\\d{4}$");
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)

--- a/backend/src/main/java/woorifisa/goodfriends/backend/report/application/ReportService.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/report/application/ReportService.java
@@ -48,7 +48,7 @@ public class ReportService {
         if(duplicateReport(productId, loginUser.getId()))
             throw new AlreadyReportedException();
 
-        User foundUser = userRepository.getByUserId(loginUser.getId());
+        User foundUser = userRepository.getById(loginUser.getId());
         Product productCreator = productRepository.getByProductIdAndUserId(productId);
         Report newReport = createReport(foundUser, productCreator, request);
         reportRepository.save(newReport);

--- a/backend/src/main/java/woorifisa/goodfriends/backend/user/application/UserService.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/user/application/UserService.java
@@ -32,7 +32,7 @@ public class UserService {
 
     @Transactional
     public void updateProfileImageToS3(final Long userId, final MultipartFile multipartFile) throws IOException {
-        User foundUser = userRepository.getByUserId(userId);
+        User foundUser = userRepository.getById(userId);
         String updateImageUrl =  updateProfileImageToS3(multipartFile);
 
         foundUser.updateProfileImageUrl(updateImageUrl);

--- a/backend/src/main/java/woorifisa/goodfriends/backend/user/domain/UserRepository.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/user/domain/UserRepository.java
@@ -10,7 +10,7 @@ import java.util.Optional;
 
 public interface UserRepository extends JpaRepository<User, Long> {
 
-    default User getByUserId(final Long id) {
+    default User getById(final Long id) {
         return findById(id)
                 .orElseThrow(NotFoundUserException::new);
     }

